templates/Web/Pages/Vue.List/frontend/src/views/VueList.vue

```diff
@@ -5,18 +5,18 @@
     </div>
     <div class="row">
       <div class="col-12 p-0">
-        <ListForm v-model="textField" @addListItem="addListItem" />
+        <ListForm v-model="textField" @add-list-item="addListItem" />
```

Comment:
@sibille @dgomezc  not 100% sure but I think this needs to match with the $emit( ) one. I´d suggest changing the function name "addListItem" to something like in the onWarningClose, so it is clear that they are two different things.

Example:
``` add-list-item=handleAddListItem```